### PR TITLE
AzureResourceManager: Adding Private postgres db support for Artifactory and Xray templates 

### PR DESCRIPTION
**Pull request by maheshjfrog**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Title of the PR starts with installer/product name (e.g. `[ansible/artifactory]`)
- [ ] CHANGELOG.md updated
- [ ] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: AzureResourceManager/Artifactory/nested/Postgresql_deploy.json [hunk 1]
```diff
@@ -91,76 +91,136 @@
...
-          "properties": {
-            "startIpAddress": "0.0.0.0",
-            "endIpAddress": "0.0.0.0"
-          }
+      "properties": {
+        "administratorLogin": "[parameters('db_user')]",
+        "administratorLoginPassword": "[parameters('db_password')]",
+        "availabilityZone": "",
+        "Backup": {
+          "backupRetentionDays": "7",
```
Comment (danielmkn) [inline AzureResourceManager/Artifactory/nested/Postgresql_deploy.json:169]: Please, use the values supplied in the parameters. We have it above `[parameters('backupRetentionDays')]`

File: AzureResourceManager/Artifactory/nested/Postgresql_deploy.json [hunk 2]
```diff
@@ -91,76 +91,136 @@
...
+          "geoRedundantBackup": "Disabled"
```
Comment (danielmkn) [inline AzureResourceManager/Artifactory/nested/Postgresql_deploy.json:170]: same here.

File: AzureResourceManager/Artifactory/nested/Postgresql_deploy.json [hunk 3]
```diff
@@ -91,76 +91,136 @@
...
-            "collation": "[parameters('databases').properties[0].collation]"
-          }
+        "highAvailability": {
+          "mode": "Disabled",
+          "standbyAvailabilityZone": ""
+        },
+        "Storage": {
+          "StorageSizeGB": "128"
+        },
+        "version": "11",
```
Comment (danielmkn) [inline AzureResourceManager/Artifactory/nested/Postgresql_deploy.json:179]: Use parameters, where possible, please.

**Comment by gitta-jfrog** [comment]
This solution has been deprecated.

**Pull request closed without merging**